.kokoro/build.sh

```diff
@@ -32,6 +34,10 @@ gcloud config set project $GCLOUD_PROJECT
 
 # Start functions emulator, if appropriate
 if [[ $PROJECT == functions/* ]]; then
+  export FUNCTIONS_LOG_PATH=$(pwd)/logs/cloud-functions-emulator.log
+  npm install -g @google-cloud/functions-emulator
```

Comment:
Is there a way we could avoid global installs? Would `npx` solve that for us?

Reply:
Perhaps - however, these containers are (AFAIK) ephemeral. Is making (and potentially debugging) this change a good use of time?

Reply:
If anybody wants to reproduce this locally, they might end up with global installs which is not nice. Also, problem caused by globally installed dependencies are hard to debug. 

It should be enough to delete the `npm i -g` line and further down call: 
`npx  functions-emulator start` (instead of `functions-emulator start`). 

Let's land it as is to figure out if it works. And then do another revision to get rid of the global installs.